src/test/scala/com/cognite/sdk/scala/v1/GreenfieldDataPointsTest.scala

```diff
@@ -0,0 +1,20 @@
+package com.cognite.sdk.scala.v1
+
+import java.util.UUID
+
+import com.cognite.sdk.scala.common.{DataPointsResourceBehaviors, SdkTest}
+
+class GreenfieldDataPointsTest extends SdkTest with DataPointsResourceBehaviors[Long] {
+  override def withTimeSeriesId(testCode: Long => Any): Unit = {
+    val timeSeriesId = greenfieldClient.timeSeries.createFromRead(
+      Seq(TimeSeries(name = s"data-points-test-${UUID.randomUUID().toString}"))
+    ).head.id
+    try {
+      val _ = testCode(timeSeriesId)
+    } finally {
+      greenfieldClient.timeSeries.deleteByIds(Seq(timeSeriesId))
+    }
+  }
+
```

Comment:
Arguably we should also have a test that would fail if greenfield wasn't used, for example by retrieving the first asset from `client` and ensuring that attempting to retrieve that id using `greenfieldClient.assets.retrieveById` fails, since this test would pass even if the `baseUrl` was ignored.